website/addons/osfstorage/tests/test_views.py

```diff
@@ -677,3 +678,42 @@ def test_remove_nonexistent_tag(self):
         res = self.app.delete_json(url, {'tag': 'Chance'}, auth=self.user.auth, expect_errors=True)
         assert_equal(res.status_code, 400)
         assert_equal(res.json['status'], 'failure')
+
+    @mock.patch('website.files.models.osfstorage.OsfStorageFile.add_tag_log')
```

Comment:
Rather than mocking, why not test that the most recent log for the project is what you expect it to be (e.g. make assertions about its actions, it's params, etc).